crates/bws/Dockerfile

```
@@ -0,0 +1,15 @@
+ARG ARCH=
+FROM ${ARCH}rust:1.73 as builder
+RUN apt-get update && \
+  apt-get install --no-install-recommends -y \
+  ca-certificates \
+  && rm -rf /var/lib/apt/lists/*
+COPY ../../ /app
+WORKDIR /app/crates/bws
+RUN cargo build --release
+FROM ${ARCH}debian:bookworm-slim as runner
+WORKDIR /usr/local/bin
+COPY --from=builder /app/target/release/bws .
+COPY --from=builder /etc/ssl/certs /etc/ssl/certs
```

I think it's clearer which stage is active if there is a newline between the stages.

Is it possible to remove the WORKDIR instruction? I think it's more obvious what's going on if the copy includes the full path. I'm also curious if `/usr/local/bin` is the correct path for the binary, I've put it in the docker root in other projects I've worked it. Admittedly they haven't been CLI's though.

```suggestion
RUN cargo build --release

FROM ${ARCH}debian:bookworm-slim as runner
COPY --from=builder /app/target/release/bws /usr/local/bin/bws
COPY --from=builder /etc/ssl/certs /etc/ssl/certs
```

---

Anywhere in `$PATH` is fine: `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`. I usually install stuff to `/usr/local/bin` because it's where binaries that aren't included with the distribution would typically go (aside from `/opt`, which isn't in `$PATH` by default).

Copying it to `/` would mean that if you overrode the entrypoint for an interactive session, you'd have to specify the full path to the binary since it wouldn't be in `$PATH`.